### PR TITLE
Try to fix ImportDumpJob

### PR DESCRIPTION
**Pull request by Universal-Omega**

### CODE REVIEW

File: includes/Jobs/ImportDumpJob.php [hunk 1]
```diff
@@ -167,9 +173,9 @@ public function run(): bool {
 			$this->importDumpHookRunner->onImportDumpJobAfterImport( $filePath, $this->importDumpRequestManager );
 		} catch ( Throwable $e ) {
 			MWExceptionHandler::rollbackPrimaryChangesAndLog( $e );
-			$this->setLastError( $this->getLogMessage( $e ) );
+			$this->jobError = $this->getLogMessage( $e );
 			$this->notifyFailed();
-			return false;
```
Comment (paladox) [inline includes/Jobs/ImportDumpJob.php:172]: Should be return false as it failed.

File: includes/Jobs/ImportDumpJob.php [hunk 2]
```diff
@@ -112,7 +117,8 @@ public function run(): bool {
 		// @phan-suppress-next-line SecurityCheck-PathTraversal False positive
 		$importStreamSource = ImportStreamSource::newFromFile( $filePath );
 		if ( !$importStreamSource->isGood() ) {
-			$this->setLastError( "Import source for {$filePath} failed" );
+			$this->jobError = "Import source for {$filePath} failed";
+			$this->setLastError( $this->jobError );
 			$this->notifyFailed();
 			return true;
```
Comment (paladox) [inline includes/Jobs/ImportDumpJob.php:123]: Should be return false.

File: includes/Jobs/ImportDumpJob.php (hunk 1 above)
Comment (Universal-Omega) [reply includes/Jobs/ImportDumpJob.php:172]: But we handle failed manually, we don't want the job to think it failed here even if it actually did.

File: includes/Jobs/ImportDumpJob.php (hunk 2 above)
Comment (Universal-Omega) [reply includes/Jobs/ImportDumpJob.php:123]: Same as above comment

**Pull request merged**